test/e2e/data/shared/v1beta1_provider/metadata.yaml

```diff
@@ -8,7 +8,10 @@
 apiVersion: clusterctl.cluster.x-k8s.io/v1alpha3
 releaseSeries:
   - major: 1
-    minor: 0
+    minor: 11
+    contract: v1beta1
+  - major: 1
+    minor: 12
```

Comment:
Are we using 1.12 for any tests at the moment or is this mostly anticipating #4359?

Reply:
Yes, and also because it seems easier to understand when doing a release (last two supported versions + the next one, as opposed to "the older supported version + the next one).